src/System.IO/ref/System.IO.cs

```diff
@@ -186,6 +186,9 @@ public partial class MemoryStream : System.IO.Stream
         public override long Position { get { throw null; } set { } }
         public override System.IAsyncResult BeginRead(byte[] buffer, int offset, int count, System.AsyncCallback callback, object state) { throw null; }
         public override System.IAsyncResult BeginWrite(byte[] buffer, int offset, int count, System.AsyncCallback callback, object state) { throw null; }
+#if netcoreapp11
```

Comment:
In `src\System.IO\ref\System.IO.csproj`, change:

`<NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>`

to 

`<NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>`.

It is necessary to pickup your earlier change.